pkg/policies/opa/rego/k8s/kubernetes_service/cve_2020_8554/ensurePrivateIP.rego

```diff
@@ -0,0 +1,16 @@
+package accurics
+
+{{.prefix}}{{.name}}{{.suffix}}[service.id] {
+    service := input.{{.resource_type}}[_]
+    service.config.kind == "Service"
```

Comment:
Looks like this check is unnecessary--kubernetes_service resource_type implies this kind

Reply:
I see, will remove that check